README.md

```diff
@@ -218,6 +218,9 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [Quantum Music Composer for IBM Q](https://github.com/JavaFXpert/quantum-toy-piano-ibmq) - Compose and perform quantum music with IBM Q.
 - [Quantum Music Composer for Rigetti](https://github.com/JavaFXpert/quantum-toy-piano) - Compose and perform quantum music with Rigetti's Forest.
 
+**JavaScript**
+- [Quantum Game with Photons](https://github.com/stared/quantum-game) - a puzzle game with real quantum mechanics in a browser
```

Comment:
Please adhere to the style guide outlined in CONTRIBUTING.MD. No 'a' or 'the' at the beginning, capitalize the first letter and end it with a period. Once you fixed that I'm happy to merge! Thanks.

Reply:
I thought it was only about the title, not desc. Updated this PR. Let me know if it's fine.